src/actions/bloodlinesActions.js

```diff
@@ -1,307 +1,120 @@
-	return {
-		type: HANDLE_GET_CONTENT,
-		payload
-	};
-}
-function errorGetContent(id, err) {
-	return {
-		type: ERROR_GET_CONTENT,
-		id,
-		err
-	};
+export function getAllTriggers(offset, limit, reset) {
+	return handlePagedRequest(TRIGGERS, TRIGGER_URL+"?offset="+offset+"&limit="+limit, "GET", offset, limit);
 }
 
-function sendCreateContent(content) {
-	return {
-		type: CREATE_CONTENT,
-		content
-	};
-}
-function handleCreateContent(payload) {
-	return {
-		type: HANDLE_CREATE_CONTENT,
-		payload
-	};
-}
-function errorCreateContent(content, err) {
-	return {
-		type: ERROR_CREATE_CONTENT,
-		content,
-		err
-	};
-}
 export function createContent(body) {
-	return dispatch => {
-		dispatch(sendCreateContent(body));
-
-		return fetch(CONTENT_URL, {
-			method: "POST",
-			body: JSON.stringify(body)
-		}).then((res) => {
-			return res.json();
-		}).then((json) => {
-			if (json.error || !json.success) {
-				dispatch(errorCreateContent(body, json.message))
-				return;
-			}
-
-			dispatch(handleCreateContent(json))
-		}).catch((err) => {
-			dispatch(errorCreateContent(body, err))
-		});
-	}
+	return handleRequest(CONTENT_URL, "POST", body);
 }
 
-function sendDeleteContent(id) {
-	return {
-		type: DELETE_CONTENT,
-		id
-	};
+export function deleteContent(id) {
+	return handleRequest(CONTENT_URL+"/"+id,"DELETE")
 }
-function handleDeleteContent(payload) {
-	return {
-		type: HANDLE_DELETE_CONTENT,
-		payload
-	};
+
+export function createTrigger(body) {
+	return handleRequest(TRIGGER_URL, "POST", body);
 }
-function errorDeleteContent(id, err) {
-	return {
-		type: ERROR_DELETE_CONTENT,
-		id,
-		err
-	};
+
+export function deleteTrigger(id) {
+	return handleRequest(TRIGGER_URL+"/" + id, "DELETE");
 }
-export function deleteContent(id) {
-	return dispatch => {
-		dispatch(sendDeleteContent(id));
 
-		return fetch(CONTENT_URL+"/"+id, {
-			method: "DELETE"
+function handlePagedRequest(item, url, type, offset, limit) {
+	return dispatch => {
+		return fetch(url, {
+			method: type
 		}).then((res) => {
 			return res.json();
 		}).then((json) => {
 			if (json.error || !json.success) {
-				dispatch(errorDeleteTrigger(id, json.message))
+				dispatch(errorPaged(item, offset, limit, json.message));
 				return;
 			}
 
-			dispatch(handleDeleteContent(json))
+			dispatch(handlePaged(item, json, offset, limit))
 		}).catch((err) => {
-			dispatch(errorDeleteContent(id, err))
+			dispatch(errorPaged(item, offset, limit, err))
 		});
 	}
 }
 
-function sendGetAllTriggers(offset, limit) {
-	return {
-		type: GET_ALL_TRIGGERS,
-		offset,
-		limit
-	};
-}
-function handleGetAllTriggers(payload, limit, reset) {
-	return {
-		type: HANDLE_GET_ALL_TRIGGERS,
-		payload,
-		limit,
-		reset
-	};
-}
-function errorGetAllTriggers(offset, limit, err) {
-	return {
-		type: ERROR_GET_ALL_TRIGGERS,
-		offset,
-		limit,
-		err
-	};
-}
-export function getAllTriggers(offset, limit, reset) {
+function handleRequest(url, type, body) {
```

Comment:
This handles any request other than a paged one. making everything else just url construction which was the only difference in the functions before.